src/test/java/com/backend/blooming/stamp/presentation/StampControllerTest.java

```diff
@@ -157,4 +161,63 @@ class StampControllerTest extends StampControllerTestFixture {
+        // given
+        given(tokenProvider.parseToken(액세스_토큰_타입, 액세스_토큰)).willReturn(사용자_토큰_정보);
+        given(userRepository.existsByIdAndDeletedIsFalse(사용자_토큰_정보.userId())).willReturn(true);
+        given(stampService.readAllByGoalId(유효한_골_아이디, 사용자_토큰_정보.userId())).willReturn(유효한_스탬프_목록_dto);
+
+        // when & then
+        mockMvc.perform(get("/stamps/{goalId}", 유효한_골_아이디)
+                .header("X-API-VERSION", 1)
+                .header(HttpHeaders.AUTHORIZATION, 액세스_토큰)
+        ).andExpectAll(
+                status().isOk(),
+                jsonPath("$.stamps.1.[0].userId", is(유효한_스탬프_목록_응답.stamps().get(1).get(0).userId()), Long.class),
+                jsonPath("$.stamps.1.[0].name", is(유효한_스탬프_목록_응답.stamps().get(1).get(0).name()), String.class),
+                jsonPath("$.stamps.1.[0].color", is(유효한_스탬프_목록_응답.stamps().get(1).get(0).color()), String.class),
+                jsonPath("$.stamps.1.[0].message", is(유효한_스탬프_목록_응답.stamps().get(1).get(0).message()), String.class),
+                jsonPath("$.stamps.1.[0].day", is(유효한_스탬프_목록_응답.stamps().get(1).get(0).day()), Integer.class),
+                jsonPath("$.stamps.2.[0].userId", is(유효한_스탬프_목록_응답.stamps().get(2).get(0).userId()), Long.class),
+                jsonPath("$.stamps.2.[1].userId", is(유효한_스탬프_목록_응답.stamps().get(2).get(1).userId()), Long.class)
+        ).andDo(print()).andDo(restDocs.document(
+                pathParameters(parameterWithName("goalId").description("스탬프를 조회할 골 아이디")),
+                requestHeaders(
+                        headerWithName("X-API-VERSION").description("요청 버전"),
+                        headerWithName(HttpHeaders.AUTHORIZATION).description("액세스 토큰")
+                ),
+                responseFields(
+                        fieldWithPath("stamps.1.[].userId").type(JsonFieldType.NUMBER).description("스탬프를 찍은 사용자 아이디"),
+                        fieldWithPath("stamps.1.[].name").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 이름"),
+                        fieldWithPath("stamps.1.[].color").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
+                        fieldWithPath("stamps.1.[].message").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
+                        fieldWithPath("stamps.1.[].day").type(JsonFieldType.NUMBER).description("스탬프를 찍은 날짜"),
+                        fieldWithPath("stamps.2.[].userId").type(JsonFieldType.NUMBER).description("스탬프를 찍은 사용자 아이디"),
+                        fieldWithPath("stamps.2.[].name").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 이름"),
+                        fieldWithPath("stamps.2.[].color").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
+                        fieldWithPath("stamps.2.[].message").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
+                        fieldWithPath("stamps.2.[].day").type(JsonFieldType.NUMBER).description("스탬프를 찍은 날짜")
+                )
```

Comment:
동일한 내용을 두 번 작성할 필요는 없다고 생각하시는 데 어떻게 생각하시나요?
1, 2가 아니라 여기도 *로 표현해 줘도 괜찮지 않나 싶습니다!
저는 아래와 같은 형태를 생각해 봤습니다!

```suggestion
                        fieldWithPath("stamps").type(JsonFieldType.OBJECT).description("날짜별 스탬프 목록"),
                        fieldWithPath("stamps.*").type(JsonFieldType.ARRAY).description("특정 날짜에 대한 스탬프 배열"),
                        fieldWithPath("stamps.*.[].userId").type(JsonFieldType.NUMBER).description("스탬프를 찍은 사용자 아이디"),
                        fieldWithPath("stamps.*.[].name").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 이름"),
                        fieldWithPath("stamps.*.[].color").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
                        fieldWithPath("stamps.*.[].message").type(JsonFieldType.STRING).description("스탬프를 찍은 사용자 프로필 색상코드"),
                        fieldWithPath("stamps.*.[].day").type(JsonFieldType.NUMBER).description("스탬프를 찍은 날짜")
```

Reply:
*로 처리하는 방법을 몰랐네요..! 이렇게 해보겠습니다. 감사합니다!